outlines/data-loading.md

```diff
@@ -0,0 +1,33 @@
+# Data Loading and Management
+
+1. Publications + Subscriptions
+  1. What are they? - compare REST endpoint
+  2. How do they work? - talk about briding collections
+  3. What's a pub / what's a sub
+2. Defining a publication on the server
+  1. Rules around what arguments it should take
+  2. Where should it go? (which package -- depends on universality)
+3. Subscribing on the client
+  1. Subscriptions should be initiated by templates/components that need the data
+  2. Global required data should be subscribed by an always there "layout" template
+  3. Retrieve the data from the sub at the same point as subscribing, pass down and filter via `cursor-utils`
```

Comment:
Yes, that's what I'm basically saying. It's probably not a hard and fast rule (e.g. `Meteor.user()`), but a good rule of thumb.

Cross talk between subscriptions? At this point I think it's just "use the same query to get data out that you used to put it in". Something like @sachag's package would obviously help with this.